app/src/main/java/at/aau/edu/wizards/gameModel/GameModelBoardInterface.kt

```diff
@@ -0,0 +1,68 @@
+package at.aau.edu.wizards.gameModel
+
+interface GameModelBoardInterface {
```

Comment:
in kotlin interfaces are usually not prefixed by `Interface`. Usually this would be `GameModelBoard` and the implementation would be `GameModelBoardImpl`.

That also makes sense for repository pattern. Eg you define interface

```
interface UserRespostory {
   fun getUser(): User
}
```

implementation could look like:

`DatabaseUserRepository` or `InMemoryUserRepository` for specific cases. For default implementation it would be `UserRepositoryImpl`

Reply:
Got rid of interfaces entirely in new implementation. Noted for future implementations.